api/src/main/java/org/openmrs/api/CohortService.java

```diff
@@ -140,13 +140,16 @@
 	 *
 	 * @since 2.1
 	 * @param patient patient used to find the cohorts
-	 * @return Cohorts that contain the given patient
+	 * @param includeVoidedCohorts voided true/false whether or not to include voided cohorts
+	 * @param includeVoidedMemberships voided true/false whether or not to include voided memberships
+	 *@param asOfDate  @return Cohorts that contain the given patient
```

Comment:
@vshl fix formatting. use the IDE formatting settings as described here https://wiki.openmrs.org/display/docs/Coding+Conventions